drivers/MbedCRC.h

```diff
@@ -258,6 +258,7 @@ class MbedCRC {
      *  This API is used to perform final computation to get correct CRC value.
```

Comment:
Note: Mbed is styled incorrectly in license info at top of page. It's not within the set of changes for this commit, so I can't attach this to the correct line.

Reply:
You mean the line:
https://github.com/ARMmbed/mbed-os/blob/5b25b6643d24c94442695a858fc9a5845d5c0957/drivers/MbedCRC.h#L1

Reply:
@melwee01 What shall be fixed?

Reply:
mbed - Mbed 
It should be fixed in all the files 
https://github.com/ARMmbed/mbed-os/blob/5b25b6643d24c94442695a858fc9a5845d5c0957/drivers/AnalogIn.h#L1

https://github.com/ARMmbed/mbed-os/blob/5b25b6643d24c94442695a858fc9a5845d5c0957/drivers/BusIn.h#L1

@0xc0170 - We don't have consistent one license header across all files in repo :-(

Reply:
> @0xc0170 - We don't have consistent one license header across all files in repo :-(

Definitely. I'll add it to the docs to start the unification

Reply:
https://github.com/ARMmbed/mbed-os-5-docs/blob/development/docs/reference/contributing/guidelines/license.md#how-to-apply-apache-20-correctly

Already there, just need to do a clean-up one day !